cmd/amplifier/main.go

```diff
@@ -52,6 +56,8 @@ func parseFlags() {
 	flag.StringVarP(&clientSecret, "clientsecret", "c", defaultClientSecret, "github app clientsecret (default '"+defaultClientSecret+"')")
 	flag.StringVarP(&kafkaURL, "kafkaURL", "k", kafkaDefaultURL, "kafka URL (default '"+kafkaDefaultURL+"')")
 	flag.StringVarP(&influxURL, "influxURL", "", influxDefaultURL, "InfluxDB URL (default '"+influxDefaultURL+"')")
+	flag.StringVarP(&dockerURL, "dockerURL", "", dockerDefaultURL, "Docker URL (default '"+dockerDefaultURL+"')")
```

Comment:
For options lowercase, it's also true for kafakURL and influxURL, should have to lowercase them alo?